androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/theme/Type.kt

```diff
@@ -0,0 +1,69 @@
+package com.EENX15_22_17.digital_journal.android.ui.theme
+
+import androidx.compose.material.Typography
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+import com.EENX15_22_17.digital_journal.android.R
+
+val Roboto = FontFamily(
+    Font(R.font.roboto_regular),
+    Font(R.font.roboto_bold, FontWeight.Bold)
+)
+
```

Comment:
Are these values created by us? Otherwise we should consider extending the source, or at least link to it.

Reply:
Added link to source